rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/core/JacksonCustomizations.java

```diff
@@ -182,6 +181,97 @@ public Sw360Module() {
             @JsonProperty("id")
             abstract public String getId();
         }
+        @JsonInclude(JsonInclude.Include.NON_NULL)
+        @JsonIgnoreProperties({
```

Comment:
This is mostly duplicated code. Is there maybe a option to say, that this the list for `Project` but with additional entries? Maybe by extending the `ProjectMixin`

Reply:
Done, Thanks for the suggestion.